# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: tools/profiler/tests/xpcshell/xpcshell.toml
# Commit: 33f1358c77ca
# Full Hash: 33f1358c77caa2e9b0fc13620ba61ce1657b9b55
# Author: Adam Brouwers-Harries <abrouwersharries@mozilla.com>
# Date: 2024-04-03 09:34:09
# Regressor Bug: 1865930
# File Overlap Count: 1
# Description:
#   Bug 1865930: Add support for stopping the profiler using UNIX signals r=canaltinova,profiler-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D203146
# ==============================================================================

diff -r d34af55bcae0 -r 33f1358c77ca tools/profiler/tests/xpcshell/xpcshell.toml
--- a/tools/profiler/tests/xpcshell/xpcshell.toml	Tue Apr 02 14:03:46 2024 +0000
+++ b/tools/profiler/tests/xpcshell/xpcshell.toml	Tue Apr 02 14:27:17 2024 +0000
@@ -24,9 +24,6 @@
 ["test_feature_fileioall.js"]
 skip-if = ["release_or_beta"]
 
-# The sanitizer checks appears to overwrite our own memory hooks in xpcshell tests,
-# and no allocation markers are gathered. Skip this test in that configuration.
-
 ["test_feature_java.js"]
 skip-if = ["os != 'android'"]
 
@@ -50,6 +47,12 @@
   "socketprocess_networking",
 ]
 
+["test_feature_posix_signals.js"]
+skip-if = [
+  "ccov",
+  "os == 'win'",
+]
+
 # Native stackwalking is somewhat unreliable depending on the platform.
 #
 # We don't have frame pointers on macOS release and beta, so stack walking does not
@@ -61,6 +64,9 @@
 # For sanitizer builds, there were many intermittents, and we're not getting much
 # additional coverage there, so it's better to be a bit more reliable.
 
+# The sanitizer checks appears to overwrite our own memory hooks in xpcshell tests,
+# and no allocation markers are gathered. Skip this test in that configuration.
+
 ["test_feature_stackwalking.js"]
 skip-if = [
   "os == 'mac' && release_or_beta",